packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -30,6 +30,39 @@ Annotated component with props:
 
 Sample usage, composes `Accordion` and `AccordionItem`:
 
+```jsx
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemTrigger,
+  AccordionItemBody,
+} from '@razorpay/blade/components';
+
+<Accordion showNumberPrefix defaultExpandedIndex={0}>
+  <AccordionItem>
+    {/* With title and description defined */}
+    <AccordionItemTrigger title="Can I create linked accounts using Route?" />
+    <AccordionItemBody>
+      <Text>
+        You can use Razorpay Route from the Dashboard or using APIs to transfer money to customers.
+      </Text>
+    </AccordionItemBody>
+  </AccordionItem>
+  {/* With custom slot for trigger */}
+  <AccordionItem>
+    <AccordionItemTrigger>
+      <div>Trigger Slot</div>
```

Comment:
In this case the props passed to `AccordionItemTrigger` will be ignored?

Reply:
if you add `title`, it renders above the slot. This is configured in our BaseHeader